public/mobile-app/src/routes/notifications/+page.svelte

```diff
@@ -16,12 +19,14 @@ onMount(async () => {
   }
 
   notifications = await retrieveNotifications()
+  notificationEventsSocket(async (event) => {
+    notifications = await retrieveNotifications()
+  })
 })
 
 const markNotificationAsRead = async (event, notificationId) => {
   event.preventDefault()
   let result = await readNotification(notificationId)
-  notifications = await retrieveNotifications()
```

Comment:
Don't we still need to retrieve the notifications on the app startup? Otherwise we won't have the backlog of notifications until a new one arrives? Am I missing something?

Reply:
This was to update notification list when a notification is marked as read.
Now we get a message from the socket on a notification update.